doc/examples/pulsating-sphere-displacement.py

```diff
@@ -0,0 +1,72 @@
+"""
+Particle displacement of a pulsating sphere.
+"""
```

Comment:
```
$ python3 -m pydocstyle pulsating-sphere-displacement.py 
pulsating-sphere-displacement.py:1 at module level:
        D200: One-line docstring should fit on one line with quotes (found 3)
```

Reply:
This one is fixed, thank you.

Reply:
No it wasn't, sorry.
I wasn't aware of `pydocstyle` and have only checked
```
$ pycodestyle
```
Thanks.